PinpointKit/PinpointKit/Sources/Core/Configuration.swift

```diff
@@ -32,7 +32,7 @@ public struct Configuration {
     }
     
     /// A feedback collector that obtains the feedback to send.
-    let feedbackCollector: FeedbackCollector
+    var feedbackCollector: FeedbackCollector
```

Comment:
Could you merge in the latest develop and then change this back to a let?

Reply:
done